spec/services/attribute_asserter_spec.rb

```diff
@@ -778,4 +832,8 @@
       it_behaves_like 'unverified user'
     end
   end
+
+  def get_asserted_attribute(user, attribute)
+    user.asserted_attributes[attribute][:getter].call(nil)
```

Comment:
(nothing needed for this PR, mentioning)

ok so I can't believe I am figuring this out today, but I think it's a really weird pattern the way we assign the transient `asserted_attributes` to the user object like this, I would love to live in a world where we refactor how we pass the user attributes around, but that seems like not the right idea for this PR right this moment

Reply:
Right, I was going to dig into why we have this strange pattern if I had some time at the end.